lib/chat/content/files.ex

```diff
@@ -10,10 +10,14 @@ defmodule Chat.Content.Files do
   def delete({key, _secret}), do: delete(key)
   def delete(key), do: Db.delete(db_key(key))
 
-  def add(data) do
-    key = UUID.uuid4()
+  def add([key, raw_secret, _, _, _, _] = data) do
+    secret = Base.decode64!(raw_secret)
```

Comment:
Should this be `url_decode`?

Reply:
No, this is the same as this:
https://github.com/Buckitup-chat/chat/blob/77980292a8b0362c37edab76df1b42088cd441c3/lib/chat_web/controllers/file_controller.ex#L40-L44